showmehow/lessons.json

```diff
@@ -126,7 +126,7 @@
                 "task": "Gibberish again. Nothing bad will happen, I promise!",
                 "effects": {
                     "success": {
-                        "reply": "See? Harmless. Don't be afraid to try new things. You can either keep typing gibberish or you can enter 'showmehow' to head back to the main menu.",
+                        "reply": "See? Harmless. Don't be afraid to try new things. You can either keep typing gibberish or you can enter 'showmehow' to the top-level of this tutorial.",
```

Comment:
"You can either keep typing gibberish or you can enter 'showmehow' to the top-level of this tutorial." Hmm, can that be instead: "You can either keep typing gibberish or type ‘showmehow’ to return to the top-Level of this tutorial.”